src/gui/folder.cpp

```diff
@@ -46,6 +45,9 @@
 
 namespace OCC {
 
+// Start with "sync." since this is mostly sync logic at the wrong layer
```

Comment:
The reason is that I wanted to be able to get all sync extra stuff by doing `QT_LOGGING_RULES="sync.*=true"`. But the `Folder` doesn't have much impact on the sync, so I guess `info` will be enough most of the time anyway, so changed it.